views/new_member_signup.erb

```diff
@@ -67,8 +67,8 @@
                             <label for="unl-permit-no">No</label>
                         </div>
                     </div>
-                </div>
```

Comment:
Can you remove all the trailing whitespace in that was added.

There is a setting in vscode for making the whitespace visible, I usually set it boundary so any leading and trailing whitespace is visible.

`"editor.renderWhitespace": "boundary",`

You can also do some regex find and replace to remove all the trailing whitespace ` +\n` (space is the first character) replaced with `\n`